### PR TITLE
Misc. Adjustments

### PR DESCRIPTION
**Pull request by N7Huntsman**
## Changes
Various tweaks and changes to various patches.
- Removed outdated VFWE patch. I can't find a version of the mod for 1.3 anywhere on the workshop and I'm not aware of it being updated anywhere else.
- Cleaned up the ammo patch for VWE to improve load time and resolve a load-order oddity related to VFE: Production's new benches.
- Fix Webley FMJ doing too much damage.
- Fix the 12 Gauge Charged Ion shots' label
- Increase parka's mass from 2.5kg to 3 to compensate for its thickness based on irl data
- Fixed vanilla chain shotgun's item description to reflect it being semi-automatic
- Fixed the patch folder and file name for the Useless Clothes mod
- Update the SupportedModList.
- Move around some xml files associated with the Misc. Turrets patch.
- Patched the catapult and trebuchet from Arrow Please and removed redundant costList stuff from weapon patches.
- Buffed Warcasket melee weapons' Toughness rating

## Testing

Check tests you have performed:
- [ ] Compiles without warnings
- [ ] Game runs without errors
- [ ] (For compatibility patches) ...with and without patched mod loaded
- [ ] Playtested a colony (specify how long)

### CODE REVIEW

File: Defs/Ammo/Advanced/12GaugeCharged.xml [hunk 1]
```diff
@@ -139,8 +139,6 @@
       </secondaryDamage>
       <armorPenetrationSharp>8</armorPenetrationSharp>      
       <armorPenetrationBlunt>14.4</armorPenetrationBlunt>
-      <pelletCount>6</pelletCount>
```
Comment (Safairette) [inline Defs/Ammo/Advanced/12GaugeCharged.xml:142]: Why revert this change? I was done on purpose to differentiate more from the regular 12 gauge, as seen from #652

**Review by Safairette — DISMISSED** [review]

**Review by trikanyo — APPROVED** [review]
Changes tested

**Review by Tostov — CHANGES_REQUESTED** [review]
Regarding arrows please (continued) patch
Minor issue with tomahawks is that they still retain the close/ short/medium/long range accuracy stats in their stat block. They also appear to have no sway.

Torch statblock is borked, I'm not sure why in this instance.

**Review by Tostov — APPROVED** [review]
Tomahawks appear to now be fixed.
If torch is unable to be fixed as per info in the discord, should be all good.

**Pull request merged**